server/test/helper.js

```diff
@@ -25,10 +36,28 @@ async function build(t) {
   // different from the production setup
   const app = await helper.build(argv, config());
 
+  // clear all the db tables
+  const prisma = new PrismaClient({
+    datasourceUrl: process.env.DATABASE_URL,
+  });
+  await prisma.$connect();
+  await prisma.$transaction([
+    prisma.physician.deleteMany(),
+    prisma.hospital.deleteMany(),
+    prisma.condition.deleteMany(),
+    prisma.medication.deleteMany(),
+    prisma.allergy.deleteMany(),
+    prisma.contact.deleteMany(),
+    prisma.patient.deleteMany(),
+    prisma.invite.deleteMany(),
+    prisma.user.deleteMany(),
+  ]);
+  await prisma.$disconnect();
+
```

Comment:
Here I make sure that the db is reset to a clean/empty state before every test. Note that this _does_ delete the seeds. If we want to have access to the seeds in test, we should create/set up fixtures for them. Still need to figure out an appropriate test fixtures solution.